google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/ConnectionImplTest.java

```diff
@@ -103,6 +110,12 @@ public void commit() {
       Timestamp commitTimestamp = Timestamp.now();
       commitResponse = mock(CommitResponse.class);
       when(commitResponse.getCommitTimestamp()).thenReturn(commitTimestamp);
+      if (returnCommitStats) {
+        CommitStats stats = mock(CommitStats.class);
```

Comment:
Isn't this just a value class? You don't need to mock it.

Reply:
True, it is just a value class, but its [constructor and static create method](https://github.com/googleapis/java-spanner/blob/66c5f88b55836abf4912e1031e504ef5d2abe9c3/google-cloud-spanner/src/main/java/com/google/cloud/spanner/CommitStats.java#L32) are not accessible from here. This test class is in the `.connection` sub-package, while `CommitStats` is in `com.google.cloud.spanner`.